Lib/calendar.py

```diff
@@ -16,7 +17,10 @@
            "timegm", "month_name", "month_abbr", "day_name", "day_abbr",
            "Calendar", "TextCalendar", "HTMLCalendar", "LocaleTextCalendar",
            "LocaleHTMLCalendar", "weekheader",
-           "MONDAY", "TUESDAY", "WEDNESDAY", "THURSDAY", "FRIDAY",
+           "Weekday", "Month", "JANUARY", "FEBRUARY", "MARCH", 
+           "APRIL", "MAY", "JUNE", "JULY", 
+           "AUGUST", "SEPTEMBER", "OCTOBER", "NOVEMEMBER", "DECEMBER",
+           "MONDAY", "TUESDAY", "WEDNESDAY", "THURSDAY", "FRIDAY", 
```

Comment:
Remove trailing whitespace:

```suggestion
           "Weekday", "Month", "JANUARY", "FEBRUARY", "MARCH",
           "APRIL", "MAY", "JUNE", "JULY",
           "AUGUST", "SEPTEMBER", "OCTOBER", "NOVEMEMBER", "DECEMBER",
           "MONDAY", "TUESDAY", "WEDNESDAY", "THURSDAY", "FRIDAY",
```